src/vm/exceptionhandling.cpp

```diff
@@ -4677,6 +4687,7 @@ VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex)
         catch (PAL_SEHException& ex2)
         {
             ex = ex2;
+            ex.ContextRecord.ContextFlags &= ~CONTEXT_EXCEPTION_ACTIVE;
```

Comment:
It was, because we were not using the contents of the context, but tracking it separately using that bool.